.github/workflows/trivy.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow performs trivy check of docker config files.
+# It triggers only on pushs to devops/containers.
+# For more info: https://github.com/aquasecurity/trivy-action
+
+name: Trivy
+
+on:
+  push:
```

Comment:
Why no pre-commit (`pull_request` trigger)?

Reply:
Was not sure if we want to affect the pre-commit, updated - [55d424c](https://github.com/intel/llvm/pull/16840/commits/55d424cd48ad3a940b77ad653d94175d96c3f1c0)